tools/manifest/item.py

```diff
@@ -108,6 +88,10 @@ def id(self):
     def meta_flags(self):
         return set(urlparse(self.url).path.rsplit("/", 1)[1].split(".")[1:-1])
 
+    @property
+    def url_base(self):
+        return "/"
```

Comment:
This is blatantly wrong. But also currently doesn't break anything (uh, we should have tests!).

Reply:
Hah, seemingly this is actually a constant! (We always set it to `/` and `SourceFile` expands the full URL.) And only used by `.url`, so we can just drop this.

Reply:
Note that it's set by the manifest and doenstream projects (notably gecko and servo) have manifests which set values other than `/` e.g. https://searchfox.org/mozilla-central/source/testing/web-platform/mozilla/meta/MANIFEST.json#1768

Reply:
(That might not mean this code wasn't useless, just checking you're aware of where the url base is used).